templates/dell/index.html

```diff
@@ -1,287 +1,331 @@
+      {{
+        image (
+        url="https://assets.ubuntu.com/v1/69f7f01d-dell-servers-cropped (1).png",
         alt="",
-        width="1000",
-        height="578",
+        width="650",
+        height="327",
         hi_def=True,
         loading="lazy"
         ) | safe
       }}
     </div>
   </div>
+</section>
 
-  <div class="u-fixed-width">
-    <hr class="p-separator">
-  </div>
-
+<section class="p-strip is-shallow">
   <div class="row">
-    <div class="col-6 u-hide--medium u-hide--small u-align--center u-vertically-center">
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/0ba3e83b-machine-learning-family.png?w=345",
-          alt="",
-          width="345",
-          height="288",
-          hi_def=True,
-          loading="lazy",
-        ) | safe
-      }}
+		<hr class="is-fixed-width u-no-margin--bottom" />
+		<div class="col-6">
+			<h2>Certified hardware</h2>
+		</div>
+		<div class="col-6">
+      <p>Ubuntu certified hardware has passed our extensive testing and review process, ensuring that Ubuntu runs well out of the box, ready for your organisation. We work closely with Dell to make Ubuntu available on a wide range of its hardware.</p>
+      <p class="u-no-padding--bottom">See Dell's certified hardware</p>
+      <ul class="p-inline-list--middot">
+        <li class="p-inline-list__item">
+          <a href="/certified/laptops?q=&limit=20&vendor=Dell">Laptops</a>
+        </li>
+        <li class="p-inline-list__item">
+          <a href="/certified/desktops?q=&limit=20&vendor=Dell">Desktops</a>
+        </li>
+        <li class="p-inline-list__item">
+          <a href="/certified/servers?q=&limit=20&vendor=Dell+Technologies">Servers</a>
+        </li>
+        <li class="p-inline-list__item">
+          <a href="/certified/devices?q=&limit=20&vendor=Dell">Devices</a>
+        </li>
+      </ul>
+		</div>
+	</div>
+</section>
+
+<section class="p-strip is-shallow">
+  <div class="row u-no-padding--top">
+    <hr class="u-no-margin--bottom" />
+    <div class="col-6">
+      <h2>Reference architecture</h2>
     </div>
     <div class="col-6">
-      <h3>
-        Power your AI ambitions
-      </h3>
-      <p>
-        With Ubuntu 18.04 LTS factory installed, Dell Precision workstations deliver the power to deploy and manage cognitive technology platforms, including machine learning, artificial intelligence and deep learning. Featuring extensive memory and outstanding processors and graphics, Dell AI-ready workstations are both flexible and affordable. They're ready out-of-the-box in a certified platform with accelerated hardware so that you can work at full speed on your AI ambitions without risk and with peace of mind.
-      </p>
-      <p>
-        <a href="https://www.dellemc.com/en-gb/ai-technologies/index.htm">
-          Discover Dell's AI workstations
-        </a>
-      </p>
+      <div>
+        <div class="p-aspect-ratio-image--reference-architecture">
```

Comment:
This image does not have a background whereas the one in the section below does. Is one of these versions outdated? Seems strange to have one with a background and one without since they're so similar

Reply:
mm I can see all background images

Reply:
@carkod  This is what I meant:
![Screenshot from 2023-02-27 16-50-52](https://user-images.githubusercontent.com/17607612/221615108-a5e0df1c-6434-4a9e-b133-d258776fc182.png)

As opposed to how these look in the design where both have a background: 
![Screenshot from 2023-02-27 17-04-09](https://user-images.githubusercontent.com/17607612/221615480-edad6afa-d9b7-410e-ab29-76778b4c54cc.png)


Not sure if it's an actual problem though